.travis.yml

```diff
@@ -1,6 +1,6 @@
 sudo: required
 
-dist: trusty
+dist: xenial
```

Comment:
If I understand correctly this just upgrades our travis tests to run using a newer Linux version? Seems fine but just wanted to make sure there were no other implications.